### PR TITLE
Update settings

### PR DESCRIPTION
**Pull request by kiraware**
This PR update the settings from

``` python
PROXY_PAGINATION_PARAM = 'pager'
PROXY_PAGINATION_DEFAULT = 'rest_framework.pagination.LimitOffsetPagination'
PROXY_PAGINATION_MAPPING = {
    'cursor': 'rest_framework.pagination.CursorPagination',
}
```

to

``` python
PROXY_PAGINATION = {
    "DEFAULT": "rest_framework.pagination.LimitOffsetPagination",
    "MAPPING": {
        "cursor": "rest_framework.pagination.CursorPagination",
    },
    "PARAM": 'pager',
}
```

And, i need your help to update `tox.ini` and `requirements.txt` in order to update `drf-proxy-pagination` dependencies. Basically this update all the required dependencies.

### CODE REVIEW

File: proxy_pagination/__init__.py [hunk 1]
```diff
@@ -1,20 +1,19 @@
-from django.conf import settings
-
 from rest_framework.pagination import BasePagination
 from rest_framework.settings import import_from_string
 
+from proxy_pagination.app_settings import api_settings
+
 __version__ = '0.2.0'
```
Comment (kiraware) [inline proxy_pagination/__init__.py:6]: We need to update this version.

File: requirements.txt [hunk 2]
```diff
@@ -3,13 +3,14 @@ Django>=1.8
 djangorestframework>=3.1
 
 # Test requirements
```
Comment (kiraware) [inline requirements.txt:5]: Update dependencies

File: tests/test_proxy_pagination.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import json
+import six
```
Comment (kiraware) [inline tests/test_proxy_pagination.py:2]: `six` is not in recent `django` anymore. So we need to install it manually.

File: tests/urls.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-from django.conf.urls import url, include
+from django.urls import include, re_path
```
Comment (kiraware) [inline tests/urls.py:1]: `url` is deprecated,  use `path` or `re_path` instead.

**Review by kiraware — COMMENTED** [review]
@tuffnatty need your suggestions sir!

**Comment by tuffnatty** [comment]
Thank you for your contribution. I like it mostly. However, I am a bit concerned about the lack of both backward compatibility or a clear and early failure when old configuration is encountered.

**Comment by kiraware** [comment]
Yes. i like about upgrading this packages. But i had little knowledge about the tox and backward compatibility. I think i need your suggestions about add more tox testing, so that we really know on which `Django` and `django-rest-framework` version this package might works well. I really hope that you can commit on this PR regarding tox and backward compatibility sir. Thanks for your attention!

**Pull request closed without merging**